ecocode-rules-specifications/src/main/rules/EC475/EC475.json

```diff
@@ -0,0 +1,11 @@
+{
+  "title": "Optimize Database Queries",
```

Comment:
Hi, for me title is quite wrong, like class names above, because you deal with database queries only for relational databases (and not NOSql databases for example). Thus, could you add "SQL" word in title, class names, descriptions, documentation, etc ... please

also, there are some lacks if you check DoD list https://github.com/green-code-initiative/ecoCode-common/blob/main/doc/starter-pack.md#definition-of-done-of-a-pr - example : 
- update RULES.md
- update CHANGELOG.md
- add test in real test repository
- ...